internal/espresso/config.go

```diff
@@ -60,6 +60,9 @@ func FromFile(cfgPath string) (Project, error) {
 	}
 	p.ConfigFilePath = cfgPath
 
+	p.Espresso.App = os.ExpandEnv(p.Espresso.App)
+	p.Espresso.TestApp = os.ExpandEnv(p.Espresso.TestApp)
```

Comment:
@valentyn88 what happens if an app it's defined in the config.yml and you try to expand a non-defined environment variable?

Reply:
If we try to use an undefined environment variable in config.yml as a result `app` will be an empty string and we will stop on the validation step because we expect to have .apk file.